### DISCUSSION TITLE
Gigabyte not showing all fans

### DISCUSSION THREAD

**Discussion by BodhiDank** [Rem0o/FanControl.Releases#2711] [Q&A]
Hey, recently I built a new pc with a gigabyte aorus b650 pro x usb 4 mobo. Problem is when trying to control the fans with fan control it only shows 4 out of the 6 connected fans. All the fans work normally on the gigabyte app, just not in fancontrol. Any clue how to fix this?

**Comment by Rem0o** [comment]
Try enabling InpOut in sensor sources.  Gigabyte boards often use 2 super io chips, fan channels being splitted between the two.  Accessing that 2nd chip is non standard.

**Reply by BodhiDank** [reply]
I already tried doing that but that isn’t changing anything.

**Reply by Rem0o** [reply]
Did you run the assisted setup again?

**Comment by Berzerker** [comment]
I'm also having this issue, InpOut is enabled and I've run the assisted setup again. I'm using an Aorus X670E Pro X
